driver/normalizer/normalizer.go

```diff
@@ -136,7 +135,8 @@ var Normalizers = []Mapping{
 	)),
 	MapSemantic("StringLiteral", uast.String{}, MapObj(
 		Fields{
-			{Name: "value", Op: singleQuote{Var("val")}},
+			{Name: "normValue", Op: Var("val")},
+			{Name: "value", Drop: true, Op: singleQuote{Any()}}, // only used in Annotated
```

Comment:
```suggestion
			{Name: "value", Drop: true, Op: Any()}, // only used in Annotated
```
No need to unquote if you drop it anyway.

Reply:
At first, I did exactly this BUT then we'll loose the `format: single` inside the Semantic node.

So I have [changed the impl of `singleQuote`](https://github.com/bblfsh/javascript-driver/pull/81/files#diff-da412cf66633e782d4653e9a0a1b54bdR380) to become no-op that just _detects_ the quotation and [matches those literals first](https://github.com/bblfsh/javascript-driver/pull/81/files#diff-da412cf66633e782d4653e9a0a1b54bdR143) - that way we can still use this signal.

Reply:
Sorry, missed that part. Makes total sense now, but it needs a comment.

As an alternative, you can drop the `Construct` method, remove the sub-operation from it and use this custom transform as a filter: `Check(singleQuote{}, Any())`.

Reply:
Addressed in fc57c535f68cff874cad98a0bcd3088c7dc6d200 - could you please check if I got that right?